src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,11 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
```

Comment:
Might be a good idea to leave this but keep it marked as deprecated in favor of the new field. You could name the new one as SKRIPT_PREFIX_MESSAGE or something

Reply:
Might be good to keep it for doing what kiip asked about making a full back definition this could be used for that

Reply:
Sure - not sure if Language/Message has a getOrDefault method

Reply:
Yeah I don't believe it does have a getOrDefault method. Along with that the usage it wouldn't return null I believe it returns the key if it's invalid